source/client/renderer/LevelRenderer.cpp

```diff
@@ -1044,11 +1044,16 @@ void LevelRenderer::renderHitOutline(Player* pPlayer, const HitResult& hr, int i
 
 	glEnable(GL_BLEND);
 	glBlendFunc(GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA);
-	glColor4f(1.0f, 1.0f, 1.0f, 0.4f);
-	glLineWidth(10.0f);
+	glColor4f(0, 0, 0, 0.4f);
 	glDisable(GL_TEXTURE_2D);
 	glDepthMask(false);
 
+	// Maximize Line Width
+	glEnable(GL_LINE_SMOOTH);
+	int range[2];
+	glGetIntegerv(GL_SMOOTH_LINE_WIDTH_RANGE, range);
+	glLineWidth(range[1]);
```

Comment:
I probably won't pull this directly, but I will integrate your changes. Why? It seems to me that things just break, I can't break any blocks.

Also this gets the maximum line width, which is 63 on Windows. Really we should be using a line width of 1, or a line width of 2 checked against the max line width range.